src/utils/err.h

```diff
@@ -621,7 +621,7 @@ ErrRecord *_err_get_record();
     }                                                \
     _ft = _fallthrough;                              \
     _fallthrough = 0;                                \
-    _last = _record.eval == errval || _ft;           \
+    _last = (_record.eval == errval) || _ft;         \
```

Comment:
Just curious what does the parenthesis do in this context? (maybe just  a basic c-programming question...)

Reply:
They are not needed, but makes the precedence more explicit. 

While debugging, I checked whether the issue could be due to the precedence between `==` and `||` operators. Added the parenthesis to avoid being unsure again